redfish-core/lib/account_service.hpp

```diff
@@ -2904,14 +2929,20 @@ inline void requestAccountServiceRoutes(App& app)
         "/redfish/v1/AccountService/Accounts/<str>/Actions/ManagerAccount.GenerateSecretKey")
         // Only the user is authorized to configure or set up MFA for their own
         // account.
-        .privileges(redfish::privileges::postManagerAccount)
+        // TODO this privilege should be using the generated endpoints, but
+        // because of the special handling of ConfigureSelf, it's not able to
+        // yet
+        .privileges({{"ConfigureUsers"}, {"ConfigureSelf"}})
         .methods(boost::beast::http::verb::post)(
             std::bind_front(handleGenerateSecretKey, std::ref(app)));
 
     BMCWEB_ROUTE(
         app,
         "/redfish/v1/AccountService/Accounts/<str>/Actions/ManagerAccount.VerifyTimeBasedOneTimePassword")
-        .privileges(redfish::privileges::postManagerAccount)
+        // TODO this privilege should be using the generated endpoints, but
+        // because of the special handling of ConfigureSelf, it's not able to
+        // yet
+        .privileges({{"ConfigureUsers"}, {"ConfigureSelf"}})
```

Comment:
You have configureUsers here but then a check to make sure the user can only do it on themselves. Do we think a user should only be able to do it on themselves ? if so, then lets remove ConfigureUsers. 
Do we think a user with ConfigureUsers should be able to do this for all users? Then lets adjust those checks on line 2773/ 2849 (make it look like https://github.com/ibm-openbmc/bmcweb/blob/bef8c80e3eca9dacbd7bd8ac127c5aa0f6a8d44f/redfish-core/lib/account_service.hpp#L2222C1-L2239C6 ) 
 
```
    bool userSelf = (req.session != nullptr &&
                     username == req.session->username);
    if (!userSelf)
    {
        BMCWEB_LOG_WARNING("Insufficient Privilege");
        messages::insufficientPrivilege(asyncResp->res);
        return;
    }
```